Userland/Libraries/LibJS/Parser.h

```diff
@@ -29,6 +29,7 @@ struct FunctionNodeParseOptions {
         IsGetterFunction = 1 << 3,
         IsSetterFunction = 1 << 4,
         IsArrowFunction = 1 << 5,
+        IsGeneratorFucntion = 1 << 6,
```

Comment:
```suggestion
        IsGeneratorFunction = 1 << 6,
```

Reply:
lol I looked at this for a solid 20 seconds, wondering what that was about